src/PrestaShopBundle/Resources/config/services/bundle/services.yml

```diff
@@ -106,3 +106,6 @@ services:
     prestashop.core.product.csv_exporter:
         class: PrestaShop\PrestaShop\Core\Product\ProductCsvExporter
         arguments: ['@translator', '@prestashop.core.admin.data_provider.product_interface']
+
+    prestashop.doctrine.ignore_foreign_keys_platform:
```

Comment:
Why do you need to define the service here as well, if it's already defined in `config/services/common.yml` ?

Reply:
is config/services/common.yml file loaded in the Main Symfony container?

Reply:
To be checked, but it should be yes this is how the CLDR service is accessible in both front and admin

Reply:
Oh no my bad it's not exactly this, the container loads `admin/service*.yml` and `front/service*.yml` depending on the context And then the common.yml is included in each folder
So to sum it up, yes it should be loaded!